web/impact/impact/v1/helpers/program_family_helper.py

```diff
@@ -0,0 +1,28 @@
+from impact.models import ProgramFamily
+from impact.v1.helpers.model_helper import ModelHelper
+from impact.v1.metadata import (
+    OPTIONAL_STRING_TYPE,
+    PK_TYPE,
+)
+
+
+class ProgramFamilyHelper(ModelHelper):
+    MODEL = ProgramFamily
+
+    DETAIL_METADATA = {
+        "id": PK_TYPE,
+        "name": OPTIONAL_STRING_TYPE,
```

Comment:
same as before - the name appears as OPTIONAL_STRING_TYPE but is in the required keys dictionary